### ISSUE TITLE
Adding RSS Feeds as daily quotes

### ISSUE THREAD

**Issue by sojusnik** [gowong/tabtrekker#76]
Would be great to have the ability to add a RSS Feed, that will show some wise or inspirational quotes every day, like offered by many websites (e.g. [here](http://feeds.feedburner.com/quotationspage/qotd), [here](http://feeds.feedburner.com/quotationspage/mqotd) and [here](http://www.aphorismen.de/tagesspruchRSS.xml)).

**Comment by gowong** [comment]
Thanks for the suggestion!

This is low priority since the main focus of TabTrekker is for exploring locations. Inspirational quotes doesn't quite fit cleanly with that purpose.

**Comment by lamga** [comment]
I agree it is feature creep however I admit I also miss the inspirational quotes that Momentum features. Why can't TabTrekker be about exploring the human mind as well as locations?

From a functional standpoint, having the ability to insert any RSS feed/Twitter feed, be it an inspiring quotation to start your day, or news headlines or even something mundane like Git notifications would be a useful addition to a browser startup page.
